code-samples/creating-your-bot/event-handling/events/guildMemberUpdate.js

```diff
@@ -1,21 +1,48 @@
 module.exports = {
 	name: "guildMemberUpdate",
 	async execute(oldMember, newMember) {
-		console.log(oldMember, newMember);
-		// if (!interaction.isChatInputCommand()) return;
+		//console.log(oldMember, newMember);
+    
+    //var defaultHost = "https://forum.citydao.io/";
+    
+    /* 
+    / Grab object for the new role
+    / newRoleId = ID of the changed Role
+    / newRoleName = Name of the changed Role
+    */
+    let newRole = newMember.roles.cache.difference(oldMember.roles.cache).last();
+    let newRoleId = newRole.id;
+    let newRoleName = newRole.name; 
```

Comment:
use object destructiring for better variable declaration like:
`const {id:newRoleId, name:newRoleName} = newRole;`